component/core/src/main/java/io/meeds/social/databind/service/DatabindServiceImpl.java

```diff
@@ -28,11 +28,14 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.scheduling.annotation.Async;
 import org.springframework.stereotype.Service;
+import org.springframework.web.server.NotAcceptableStatusException;
```

Comment:
Should be replaced by

```suggestion
import java.lang.IllegalStateException;
```

In fact, `NotAcceptableStatusException` is an exception to use in Presentation Layer as REST response, see its javadoc:

![image](https://github.com/user-attachments/assets/8811912d-d472-433a-88bc-0f8ca7a9a288)

Reply:
Yes, thanks!